source/frontend/src/components/maps/MapView.test.tsx

```diff
@@ -2,18 +2,17 @@ import userEvent from '@testing-library/user-event';
 import axios from 'axios';
 import MockAdapter from 'axios-mock-adapter';
 import { FeatureCollection } from 'geojson';
-import React from 'react';
 
 import { Claims } from '@/constants/claims';
 import { useApiGeocoder } from '@/hooks/pims-api/useApiGeocoder';
 import { useApiProperties } from '@/hooks/pims-api/useApiProperties';
 import { IProperty } from '@/interfaces';
-import { mockParcel } from '@/mocks/filterData.mock';
+import { mockParcel2 } from '@/mocks/filterData.mock';
```

Comment:
can we come up with a better name than this? do we need both mock parcels?

Reply:
I agree, will remove